src/backend/distributed/commands/common.c

```diff
@@ -28,6 +28,8 @@
 #include "distributed/metadata/distobject.h"
 #include "distributed/multi_executor.h"
 #include "distributed/worker_transaction.h"
+#include "catalog/pg_database.h"
```

Comment:
❤️ 

I think it might be a good idea to enforce this in `citus_indent`/`uncrustify`.
I will have a look what it would take to enforce this in our styling, including a complete rewrite of all our include sections.